classes/Product.php

```diff
@@ -52,6 +52,9 @@ class ProductCore extends ObjectModel
     /** @var int default Category id */
     public $id_category_default;
 
+    /** @var array */
```

Comment:
Yes it is. Have a look at [this line in AdminImportController|https://github.com/PrestaShop/PrestaShop/blob/ecaf9938fb0a455f72a6d3ba82849bb738c17761/controllers/admin/AdminImportController.php#L1753]